alembic/versions/b6aff0a993d7_add_person_and_voicesamples.py

```diff
@@ -0,0 +1,55 @@
+"""Add person and voicesamples
+
+Revision ID: b6aff0a993d7
+Revises: 33bddba74d25
+Create Date: 2024-03-01 08:56:55.205553
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+import sqlmodel 
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'b6aff0a993d7'
+down_revision: Union[str, None] = '33bddba74d25'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # Use batch operations to support SQLite ALTER TABLE for adding constraints
+    with op.batch_alter_table('utterance', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('person_id', sa.Integer(), nullable=True))
+        batch_op.create_foreign_key('fk_utterance_person', 'person', ['person_id'], ['id'])
```

Comment:
Would it make sense to store the vector embedding of the voice here? This way, we would be able to
- Show distinct speakers in the UI even without having the Persons in the DB
- On creating a new person, easily find all the instances in the past when that person spoke by fetching all the utterances with similar enough voice embeddings